### DISCUSSION TITLE
Does the relearn theme support mathjax?

### DISCUSSION THREAD

**Discussion by roncrump** [McShelby/hugo-theme-relearn#119] [Q&A] [answered]
Hi,

Is there any built-in support for mathjax equations (or an alternative)?

Or is it a case of searching online and following on of the suggestions?

Lovely theme, by the way.

Regards,
Ron.

**Comment by McShelby** [comment]
No, this is not a native feature of this theme.

Nevertheless, you can add this to your site by using the `custom-header.html` / `custom-footer.html` partials.

**Comment by McShelby** [comment, marked as answer]
Since version 5.0.0, Mathjax support is included natively in the theme
